src/index.js

```diff
@@ -0,0 +1,148 @@
+const state = {
+
+};
+
+const setUpTemperature = () => {
+  let classType;
+  let gardenType;
+  // let temperature = state.temp
+
+  if (state.temp >= 80) {
+    classType = "red";
+    gardenType = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+  } else if ((state.temp >= 70) & (state.temp <= 79)) {
+    classType = "orange";
+    gardenType = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+  } else if ((state.temp >= 60) & (state.temp <= 69)) {
+    classType = "yellow";
+    gardenType = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+  } else if ((state.temp >= 50) & (state.temp <= 59)) {
+    classType = "green";
+    gardenType = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  } else {
+    classType = "teal";
+    gardenType = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  }
+
+  let temp = document.querySelector("#temp");
+  temp.setAttribute("class", classType);
+  temp.textContent = state.temp;
+
+  let garden = document.querySelector("#garden");
+  garden.textContent = gardenType;
+  garden.setAttribute("class", classType);
+};
+
+function Up() {
```

Comment:
```suggestion
function up() {
```
javascript function names should be camelCase 🐫